guide/english/certifications/javascript-algorithms-and-data-structures/basic-algorithm-scripting/falsy-bouncer/index.md

```diff
@@ -59,7 +86,7 @@ The `Array.prototype.filter` method expects a function that returns a `Boolean`
 
 If you found this page useful, you can give thanks by copying and pasting this on the main chat:
 
-**`Thanks @renelis @abhisekp @Rafase282 for your help with Algorithm: Falsy Bouncer`**
+**`Thanks @renelis @abhisekp @Rafase282 @merkur0 for your help with Algorithm: Falsy Bouncer`**
```

Comment:
```suggestion

```
I recommend removal of the whole line. Self-promotion is not allowed.

Reply:
"Notes for Contributors" says:
"Please add your username only if you have added any relevant main contents. (  :warning: DO NOT remove any existing usernames)"

Reply:
I think we’ll get rid of that too. Your opinions requested @RandellDawson @lynxlynxlynx

Reply:
I agree, looks odd — we have git history anyway.

Looks like it's been copy-pasted to a lot of articles: https://github.com/freeCodeCamp/freeCodeCamp/search?q=Please+add+your+username+only&unscoped_q=Please+add+your+username+only
Anyone know the source?

Reply:
Hi @merkur0 

Thanks a lot for these changes. First of all this not your fault. Apologies about the confusion. 

Looking at these instructions:

> If you found this page useful, you can give thanks by copying and pasting this on the main chat:

These are from our old wiki, and the time when we used to ask our campers/users to thank the article authors in the main chat. This would give them some brownie points via a tool that we had back then.

This has since been long deprecated and should be removed now. Here and in every article.

Thanks for the PR.

Reply:
@raisedadead 
Alright, thanks for making this clear.